gateway-js/src/loadSupergraphSdlFromStorage.ts

```diff
@@ -36,28 +43,74 @@ const { name, version } = require('../package.json');
 
 const fetchErrorMsg = "An error occurred while fetching your schema from Apollo: ";
 
-export async function loadSupergraphSdlFromStorage({
-  graphId,
-  graphVariant,
-  apiKey,
-  endpoint,
+async function submitOutOfBandReport({
+  error,
+  request,
+  response,
+  startedAt = '',
+  endedAt = '',
+  tags,
   fetcher,
 }: {
-  graphId: string;
-  graphVariant: string;
-  apiKey: string;
-  endpoint: string;
+  error: any;
+  request: Request;
+  response?: Response;
+  startedAt: string;
+  endedAt: string;
```

Comment:
I'm tempted to suggest that we perhaps keep these as native date objects until right when we ship them off (and convert them to `ISO` at that point).  Just a thought though, seems fine to keep it this way.  (String types are just common and can be more easy to mix up since TypeScript doesn't have an opaque type notion that could represent an "ISO Date string".)

Not a firm ask, just musing about myself.  This is fine.